lib/rules/max-top-level-suites.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * @fileoverview Limit the number of top-level suites in a single file
+ * @author Alexander Afanasyev
+ */
+
+var astUtil = require('../util/ast'),
+    defaultSuiteLimit = 1;
+
+module.exports = function (context) {
+    var stack = [],
+        topLevelDescribes = [],
+        suiteLimit = context.options[0] ? context.options[0] : defaultSuiteLimit;
+
+    return {
+        CallExpression: function (node) {
+            if (astUtil.isDescribe(node)) {
+                stack.push(node);
+            }
+        },
+
+        'CallExpression:exit': function (node) {
+            if (astUtil.isDescribe(node)) {
+                if (stack.length === 1) {
+                    topLevelDescribes.push(node);
+                }
+
+                stack.pop(node);
+            }
+        },
+
+        'Program:exit': function () {
+            if (topLevelDescribes.length > suiteLimit) {
+                context.report({
+                    node: topLevelDescribes[1],
```

Comment:
I guess `topLevelDescribes[1]` needs to be changed to `topLevelDescribes[suiteLimit]`, otherwise the rule would always report the second top-level describe node, right?